### PR TITLE
Guild settings

### PR DESCRIPTION
**Pull request by binaryoverload**
Added a feature to enable us to change boolean flags using reflection

### CODE REVIEW

**Comment by binaryoverload** [comment]
Waiting on #43

**Comment by weeryan17** [comment]
@binaryoverload this ready to be merged?

File: src/main/java/com/cascadebot/cascadebot/commandmeta/CommandContext.java [hunk 1]
```diff
@@ -137,6 +142,40 @@ public Double getArgAsDouble(int index) {
...
+
+    public String getUsage(ICommandExecutable command, String parent) {
+        Set<Argument> arguments = getArguments(command);
+
+        Argument parentArg = Argument.of(command.command(), command.description(), arguments);
+
+        int levels = 0;
+        for (String arg : args) {
+            levels++;
+            levels++;
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/commandmeta/CommandContext.java]: 👀

**Comment by binaryoverload** [comment]
At the moment I'm not quite liking how the argsExists works, I'll add it to the agenda

**Comment by binaryoverload** [comment]
@weeryan17

**Pull request merged**